lambda/main/constants.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+module.exports.AmazonIntent = {
+    INTENT_REQUEST: 'IntentRequest',
+    STOP: 'AMAZON.StopIntent',
+    HELP: 'AMAZON.HelpIntent',
+    CANCEL: 'AMAZON.CancelIntent',
+    YES: 'AMAZON.YesIntent',
+    NO: 'AMAZON.NoIntent',
+    FALLBACK: 'AMAZON.FallbackIntent',
+};
+
+module.exports.ElasticPathIntents = {
+    ADD_TO_CART: 'AddToCartIntent',
+    ADD_TO_WISHLIST: 'AddToWishlistIntent',
+    CHECKOUT: "CheckOutIntent",
+    DESCRIBE_LISTED_PRODUCT: 'DescribeListedProductIntent',
+    DESCRIBE_PRODUCT: 'DescribeProductIntent',
+    GET_CART: 'GetCartIntent',
+    GET_WISLIST: 'GetWishlistIntent',
+    KEYWORD_SEARCH: 'KeywordSearchIntent',
+    MOVE_TO_CART: 'MoveToCartIntent',
+    MOVE_TO_WISHLIST: 'MoveToWishlistIntent',
+    NEXT_ITEM: 'NextItemIntent',
+    PREVIOUS_ITEM: 'PreviousItemIntent',
+    REMOVE_FROM_CART: 'RemoveFromCartIntent',
+    REMOVE_FROM_WISHLIST: 'RemoveFromWishlistIntent',
+    SKU_SEARCH: 'SkuCodeSearchIntent',
+    SPECIFIC_ITEM: 'SpecificItemIntent',
+};
+
+module.exports.PROD_DESCRIPTION = 'summary';
```

Comment:
For future reference, Git will still complain about missing new-line chars at the end of files. We can clean this up later.